docs/reqs/reqs.tex

```diff
@@ -240,6 +252,12 @@ \subsection{Multirotor RPAS}
     \item The drone should have systems for low-battery alarm and emergency protocol for low-battery and loss of control radio.
 \end{enumerate}
 
+\subsection{Computing Platform}
+\begin{enumerate}[{NF.CP}.1:]
+	\item Computing Platform must be capable of performing a power-on self-test (POST) to ensure communications and video links are established
+	\item Computing Platform must keep its configuration across flight sessions, including if the battery has been detached
```

Comment:
Suggest:

...must have persistent configuration using non-volatile memory even in the case of power cut